README.md

```diff
@@ -1,2 +1,796 @@
-# mantiumclient-js
-JavaScript client for the Mantium AI API.
+
+<h3 align="center">Mantium</h3>
+
+## Table of Contents
+- [Quickstart](#quickstart)
+- [Authentication](#authentication)
+- [Installation](#installation)
+- [Usage](#usage)
+  - [Initializing](#initializing)
+  - [Auth](#auth)
+    - [Login](#login)
+    - [Logout](#logout)
+    - [Reset Password](#reset-password)
+  - [AI Methods](#ai-methods)
+    - [List Methods](#list-methods)
+  - [AI Engines](#ai-engines)
+    - [Get All AI Engines](#get-all-ai-engines)
+    - [Get Ai Engines By Provider](#get-ai-engines-by-provider)
+    - [Get Ai Engine By Name](#get-ai-engine-by-name)
+  - [Tags](#tags)
+    - [List Tags](#list-tags)
+    - [Create a Tag](#create-a-tag)
+    - [Get Tag using ID url](#get-tag-using-id-url)
+    - [Get Tag ](#get-tag)
+    - [Update Tag](#update-tag)
+    - [Delete Tag](#delete-tag)
+## Quickstart:
+Read the [getting started guide](https://developer.mantiumai.com/docs) for more information on how to use Mantium.
+## Authentication
+Make an account by visiting [app.mantiumai.com](https://app.mantiumai.com) and selecting register. Enter your email address and create a password. After you verify the email, you'll be able to sign in to the Mantium application. You'll also need your username and password to obtain a token for API use.
+
+## Installation
+You need [Node.js](https://nodejs.org/en/) installed on your computer. To install JavaScript Library please use following command
```

Comment:
Install  [Node.js](https://nodejs.org/en/) on your computer. To install JavaScript Library please use the following command

Reply:
Done